src/components/ProjectsDropdown/index.tsx

```diff
@@ -39,7 +39,7 @@ function ProjectsDropdown<T extends { projectId?: number | string } | undefined>
     if (allowUnselect) {
       options.push({
         label: unselectLabel ?? strings.NO_PROJECT,
-        value: '',
+        value: undefined,
```

Comment:
With this change in the documents list, by default, a blank value will be selected for project instead of "All", but we can change how that works to make it work with undefined

Reply:
would you mind taking that on?

Reply:
Sure! I will take it!